internal/provider/provider.go

```diff
@@ -32,6 +32,12 @@ func Provider() *schema.Provider {
 				Description: "Fully qualified domain name or IP address of the SDDC Manager",
 				DefaultFunc: schema.EnvDefaultFunc(constants.VcfTestUrl, nil),
 			},
+			"allow_unverified_tls": {
```

Comment:
Is this only for test purposes? Or is this exposed to customers as well?

Reply:
Exposed to customers as well. This is the case with the vSphere provides also:

https://github.com/hashicorp/terraform-provider-vsphere/blob/main/vsphere/provider.go#L42

My thinking was that customers may have self-signed certificates in an airgapped env, where this option is convenience.